assets/javascripts/discourse/components/chat-user-avatar.js

```diff
@@ -3,13 +3,16 @@ import discourseComputed from "discourse-common/utils/decorators";
 import { inject as service } from "@ember/service";
 
 export default Component.extend({
-  tagName: "div",
-  classNames: ["tc-presence-flair"],
-  classNameBindings: ["online"],
+  tagName: "",
```

Comment:
If we skip setting `tagName` entirely and just insert classNames: "chat-user-avatar" and remove the first div in the template, it's the same thing right?

Reply:
BTW this also means no `click(event) {}`, attribute bindings, classname bindings...